src/logic/__tests__/linkGithubIdentityTest.js

```diff
@@ -59,13 +63,12 @@ describe('linkGithubIdentity', () => {
       error.credential.accessToken,
     );
     expect(type).toBe('ACCOUNT_MIGRATION_NEEDED');
-    expect(providerId).toBe('github.com');
-    expect(accessToken).toBe('abc123');
+    expect(providerId).toBe(error.credential.providerId);
+    expect(accessToken).toBe(error.credential.accessToken);
```

Comment:
:+1: one of the best things about using factories is that we stop duplicating value literals between setup and expectations!